src/Misc/MoveToNemo.jl

```diff
@@ -0,0 +1,11 @@
+###############################################################################
+# A place to accumulate code that should eventually be moved to Nemo.jl
+###############################################################################
+
+function Hecke.numerator(f::QQPolyRingElem, parent::ZZPolyRing = Hecke.Globals.Zx)
+  g = parent()
+  ccall((:fmpq_poly_get_numerator, Nemo.libflint), Cvoid, (Ref{ZZPolyRingElem}, Ref{QQPolyRingElem}), g, f)
+  return g
+end
+
+Hecke.minpoly(a::QQBarFieldElem) = minpoly(Hecke.Globals.Qx, a)
```

Comment:
This can't be moved to Nemo as-is, it references Hecke.Globals.Qx. Similar above with `Hecke.Globals.Zx`.

... unless we also decide to move those to Nemo? Should we? @thofma @fieker

Reply:
Yes, that's true. I think this should be done, but we can discuss this elsewhere (in my upcoming issue/PR regarding this in Nemo e.g.) and not need to block this PR by this